File: platforms/swift/examples/core/shader/set_registry.swift

import FragmentColor

// Point at your own mirror of the registry
Shader.setRegistry("https://cdn.example.com/shaders/")

// Now """sdf2d/circle""" resolves to https://cdn.example.com/shaders/sdf2d/circle.wgsl
// (Skipping the actual fetch in this doctest)

File: platforms/python/examples/core/shader/set_registry.py

from fragmentcolor import Shader

# Point at your own mirror of the registry
Shader.set_registry("https://cdn.example.com/shaders/")

# Now the slug "sdf2d/circle" resolves to https://cdn.example.com/shaders/sdf2d/circle.wgsl
# (Skipping the actual fetch in this doctest)